unlock-app/src/__tests__/components/interface/GlobalErrorConsumer.test.js

```diff
@@ -151,6 +151,70 @@ describe('GlobalErrorConsumer', () => {
       const errorWrapper = rtl.render(Error)
       expect(errorWrapper.queryByText('Fatal Error')).not.toBeNull()
     })
+    describe('overrideMapping', () => {
+      it('overriding FATAL_NO_USER_ACCOUNT', () => {
+        expect.assertions(2)
+
+        const mapping = {
+          FATAL_NO_USER_ACCOUNT: () => <div>overrode</div>, // eslint-disable-line react/display-name
```

Comment:
I have seen you use a lot of disabling eslint rule. I am not necessarily against in general but I also think that if we end up doing that all the time, then there is no point in using eslint ;)

Why dud you disable this one?

Reply:
Maybe a better way to make this more clear is to write this in 2 statements?

Reply:
👍